web/surfing/src/components/Video/Video.tsx

```diff
@@ -39,37 +40,49 @@ export const Video = ({
   type,
   mimeType = "video/mp4",
   loop,
-  display = "inline",
-  preload = "auto",
-  scale = 1,
-  opacity = 1,
+  display="inline",
+  preload="auto",
+  scale=1,
+  opacity=1,
   margin = `0px 0px 0px 0px`,
-  onEnded = () => {},
-  onLoad = () => {},
+  onEnded=() => {},
+  onLoad=() => {},
   className,
-  width = "dynamic",
-  height = 900,
+  width="dynamic",
+  height=900,
+  playbackRate = 1,
   ...props
 }: IPropsVideo) => {
+  
   const classProps = className || "";
-
-  const dynamicWidth = isFirefox ? `${scale * 400}px` : `${scale * 100}%`;
-
+  
+  const dynamicWidth = isFirefox
+    ? `${scale * 400}px`
+    : `${scale * 100}%`;
+  
   let widthProp = width;
-
+  
   if (widthProp === "dynamic") {
     widthProp = dynamicWidth;
   } else if (typeof widthProp === "number") {
     widthProp = `${widthProp}px`;
   }
-
-  const heightProp = typeof height === "number" ? `${height}px` : height;
-
-  const vidRef = useRef(null);
+  
+  const heightProp = typeof height === "number"
+    ? `${height}px`
+    : height;
+  
+  const vidRef = useRef<HTMLVideoElement>(null);
   useEffect(() => {
-    vidRef.current.play();
+    vidRef.current?.play();
   });
 
+  useEffect(() => {
+    if (vidRef.current) {
+      vidRef.current.playbackRate = playbackRate;
+    }
+  }, [playbackRate, vidRef.current]);
```

Comment:
you can't put a `ref` as a useEffect dep, so this should just be `[playbackRate]`